#! /bin/sh /usr/share/dpatch/dpatch-run
## 45_link_gtk_libs.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: Link the kazehakase binary against the GTK libs.

@DPATCH@
diff -urNad kazehakase-0.4.8~/src/Makefile.am kazehakase-0.4.8/src/Makefile.am
--- kazehakase-0.4.8~/src/Makefile.am	2007-08-29 18:45:10.000000000 +0300
+++ kazehakase-0.4.8/src/Makefile.am	2008-04-03 23:35:31.000000000 +0300
@@ -109,7 +109,8 @@
  kazehakase_LDADD =			\
 	$(DBUS_GLIB_LIBS)		\
 	libkazehakase.la		\
-	$(KAZEHAKASE_RES)
+	$(KAZEHAKASE_RES)		\
+	$(GTK_LIBS)
 
 if WITH_GECKO
 kazehakase_LDFLAGS = -R$(MOZILLA_HOME)
